### DISCUSSION TITLE
gui highlight_inst

### DISCUSSION THREAD

**Discussion by gyu511** [The-OpenROAD-Project/OpenROAD#4819] [Q&A]
hello.
In the gui, highlight_inst only supports up to 8 groups, so I would like to expand it to support more groups.
So I added to HighlightGroupDialog in OpenROAD/src/gui/src/highlightGroupDialog.cpp and added color in OpenROAD/src/gui/include/gui/gui.h and finally button in OpenROAD/src/gui/ui/highlightGroupDlg.ui I also added it, but it doesn't work. What should I do?

**Comment by maliberty** [comment]
Can you send a pointer to your branch with the changes?  How does it not work?

**Comment by maliberty** [comment]
Once this works I hope you would be willing to make a PR so we can incorporate it in OR.
